src/main/groovy/de/dkfz/roddy/execution/jobs/ProcessingParameters.groovy

```diff
@@ -48,18 +54,22 @@ class ProcessingParameters implements Serializable {
         return new ProcessingParameters(parameters)
     }
 
-    String getProcessingCommandString() {
-        // Careful with this method. We had one problem where an Integer value was in one of the Collections inside the map.
-        // In this particular case, Groovy said there is something wrong with the Integer [1].
-        (parameters.asMap() as Map<String, Collection<String>>).collect { k, vs ->
-            vs.collect { v ->
-                k + ' ' + v
-            }.join(' ')
-        }.join(' ')
+    EscapableString getProcessingCommandString() {
```

Comment:
Kind of true, but generally, the type of the method is everything that counts, and the method name can at best give a hint, because everything else is unnecessarily wordy and redundant. For instance, for arguments I usually do not to add any types-suffixes to method names. For instance, in the method `fromString` the suffix is really just a big waste of letters, because Groovy is strongly typed and will anyway select the right method.

For return types the name in the method name makes sense per se. So I think something like this is needed here. So `...String` is one option.

But then, why does it *have* to be `...EscapableString`. 

1. From a semantic perspective, a `EscapableString` is nothing more than a pimped `String` and it is really easy to translate into a true `String`. 
2. Many methods returning a `String` in Groovy, actually return a `GString`.

That's not really a fair comparison (I know), because `GString` has an API that is almost identical to that of `String` -- but `EscapableString` does not have such an API.

I find `EscapableString` too clunky here. The meaning is to return some form of String-like object, and there is only one type that is returned (i.e. there is no alternative method to return a Java `String` or `GString`, or whatever). There is little room for confusion.